src/picongpu/include/particles/ionization/byField/ADK/AlgorithmADK.hpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2015 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "types.h"
+#include "simulation_defines.hpp"
+#include "particles/traits/GetAtomicNumbers.hpp"
+#include "traits/attribute/GetChargeState.hpp"
+#include "algorithms/math/floatMath/floatingPoint.tpp"
+#include "plugins/radiation/utilities.hpp"
```

Comment:
How about we decide where to put these utilities first and spare me from introducing a functor that I would remove afterwards again, anyway? This would look a little cleaner to me. 

I agree that side-including a plugin isn't cool but it just makes sense to go on with the list of existing functors for my need. I also use `util::square` and `util::cube` in my model and they are already defined. 
Afterwards we can relocate them, make them more generic and change all include paths at once.

Would that be alright with you or is it too unsexy?